etl.yaml

```diff
@@ -19,9 +19,13 @@ spec:
       configMap:
         name: useryaml
 
-    - name: host-mount
+    - name: studies-mount
       hostPath:
-        path: /Users/walsbr/aced/data_model/studies        
+        path: /Users/walsbr/aced/data_model/studies
+
+    - name: output-mount
+      hostPath:
+        path: /Users/walsbr/aced/data_model/output
```

Comment:
I'll look for possible ways to make `hostPath` path mounts relative or otherwise host-agnostic.